react/src/reactTextMask.js

```diff
@@ -19,14 +19,17 @@ export const MaskedInput = React.createClass({
   },
 
   componentDidMount() {
-    const {props, props: {value}} = this
+    this.textMaskInputElement = createTextMaskInputElement()
+    this.update()
+  },
 
-    this.textMaskInputElement = createTextMaskInputElement({inputElement: this.inputElement, ...props})
-    this.textMaskInputElement.update(value)
+  update(value) {
+    const textMaskConfig = Object.assign({inputElement: this.inputElement}, this.props)
```

Comment:
Example app probably ships with `babel-polyfill`. Not a big deal, but something to consider. Would rather stick with spread operator is all.